File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessStaticVariableIndirect_VarVarToString.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessStaticVariableIndirect_VarVarToString {
	private BackgroundService bckg;
	private String test;

	public AccessStaticVariableIndirect_VarVarToString(){
	}

	// Indirect access via static attribute and toString
		@SuppressWarnings("static-access")
		public String case7() {
			test = bckg.serviceOne.sName.toString();
			return test;
		}
}

File: testresources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessStaticVariableIndirect_VarVarToString.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessStaticVariableIndirect_VarVarToString
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;

		public void Case7()
		{
			test = bckg.serviceOne.sName.ToString();
		}
	}
}
